### PR TITLE
Add Steps component to `kontent-ai.mdx`

### PR DESCRIPTION
**Pull request by Egpereira**
# Add Steps component to `kontent-ai.mdx`

Adds the `Step` component where needed.

### CODE REVIEW

File: src/content/docs/en/guides/cms/kontent-ai.mdx [hunk 1]
```diff
@@ -529,17 +530,25 @@ If your project is using Astro's default static mode, you will need to set up a
...
 
+<Steps>
 1. Go to your project dashboard and click on **Settings**.
+
 2. Under the **Git** tab, find the **Deploy Hooks** section.
+
 3. Provide a name for your webhook and the branch you want to trigger the build on. Click **Add** and copy the generated URL.
+</Steps>
 
 Adding a webhook to Kontent.ai
```
Comment (TheOtterlord) [inline src/content/docs/en/guides/cms/kontent-ai.mdx:553]: Feels like this also needs a heading
```suggestion
##### Adding a webhook to Kontent.ai
```

**Review by TheOtterlord — APPROVED** [review]
LGTM! I have one comment regarding the surrounding context

File: src/content/docs/en/guides/cms/kontent-ai.mdx (hunk 1 above)
Comment (Egpereira) [reply src/content/docs/en/guides/cms/kontent-ai.mdx:553]: Agreed!

**Comment by sarah11918** [comment]
!coauthor

**Pull request merged**